### PR TITLE
docs: update doc links in usage doc

### PR DESCRIPTION
**Pull request by hixichen**
**why we need it**:

Links in `https://secrets-store-csi-driver.sigs.k8s.io/getting-started/usage.html` are broken.
This CR simply fix the link by referring to files in github repo.
**Special notes for your reviewer**:
N/A

### CODE REVIEW

File: docs/book/src/getting-started/usage.md [hunk 1]
```diff
@@ -47,15 +47,15 @@ foo
 
 ## [OPTIONAL] Sync with Kubernetes Secrets
 
-Refer to [Sync as Kubernetes Secret](topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
+Refer to [Sync as Kubernetes Secret](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
```
Comment (aramase) [inline docs/book/src/getting-started/usage.md:50]:
```suggestion
Refer to [Sync as Kubernetes Secret](../topics/sync-as-kubernetes-secret.md) for steps on syncing the secrets-store content as Kubernetes secret in addition to the mount.
```

We only need the relative path for the docs website to redirect to correct page.

File: docs/book/src/getting-started/usage.md [hunk 2]
```diff
@@ -47,15 +47,15 @@ foo
...
 
 ### [OPTIONAL] Set ENV VAR
 
-Refer to [Set as ENV var](topics/set-as-env-var.md) for steps on syncing the secrets-store content as Kubernetes secret and using the secret for env variables in the deployment.
+Refer to [Set as ENV var](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/set-as-env-var.md) for steps on syncing the secrets-store content as Kubernetes secret and using the secret for env variables in the deployment.
```
Comment (aramase) [inline docs/book/src/getting-started/usage.md:54]:
```suggestion
Refer to [Set as ENV var](..topics/set-as-env-var.md) for steps on syncing the secrets-store content as Kubernetes secret and using the secret for env variables in the deployment.
```

File: docs/book/src/getting-started/usage.md [hunk 3]
```diff
@@ -47,15 +47,15 @@ foo
...
 
 ## [OPTIONAL] Enable Auto Rotation of Secrets
 
-You can setup the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [Secret Auto Rotation](topics/rotation.md) for steps on enabling auto rotation.
+You can setup the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [Secret Auto Rotation](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/docs/book/src/topics/secret-auto-rotation.md) for steps on enabling auto rotation.
```
Comment (aramase) [inline docs/book/src/getting-started/usage.md:58]:
```suggestion
You can set up the Secrets Store CSI Driver to periodically update the pod mount and Kubernetes Secret with the latest content from external secrets-store. Refer to [Secret Auto Rotation](../topics/secret-auto-rotation.md) for steps on enabling auto rotation.
```

**Review by aramase — CHANGES_REQUESTED** [review]
@hixichen Thank you for the PR. Could you [sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md#how-do-i-sign)?

**Comment by aramase** [comment]
/retitle docs: update doc links in usage doc

**Comment by aramase** [comment]
@hixichen Looks like the CLA is signed now. 

Could you take a look at the comments? After those are resolved, Please squash the commits and then we can review again and merge.

File: docs/book/src/getting-started/usage.md (hunk 1 above)
Comment (hixichen) [reply docs/book/src/getting-started/usage.md]: sure, updated.

**Comment by aramase** [comment]
@hixichen Could you also squash the commits?

**Comment by hixichen** [comment]
> @hixichen Could you also squash the commits?

sorry, I will make sure that I check the reply every few hours and squash all commits in the future.

**Comment by aramase** [comment]
> sorry, I will make sure that I check the reply every few hours and squash all commits in the future.

No worries! Thank you for PR. Looks good!

**Review by aramase — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**